src/stream_item.rs

```diff
@@ -42,6 +42,18 @@ impl<T : Send + 'static> GrpcStreamWithTrailingMetadata<T> {
         GrpcStreamWithTrailingMetadata::new(stream.map(ItemOrMetadata::Item))
     }
 
+    /// Stream of items with no trailing metadata
+    pub fn stream_with_trailing<S>(stream: S, trailing: GrpcFutureSend<Metadata>) -> GrpcStreamWithTrailingMetadata<T>
+        where S : Stream<Item=T, Error=Error> + Send + 'static
```

Comment:
I think it is better to have type parameter for future here:
```
trailing: F
...
where F : Future<Item=T, Error=Error>
```

At least for consistency with other parameter.